<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TurkishIdQt</string>
  </property>
  <property name="windowIcon">
   <iconset resource="turkishidqt.qrc">
    <normaloff>:/image/TurkishIdQt.png</normaloff>:/image/TurkishIdQt.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditTC"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCheck">
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSpinBox" name="spinBoxGenerate">
        <property name="maximum">
         <number>1000000000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonRandom">
        <property name="text">
         <string>Random</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonSequental">
        <property name="text">
         <string>Sequental</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonGenerate">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEditGenerate"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>475</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="turkishidqt.qrc"/>
 </resources>
 <connections/>
</ui>
